src/NUnitFramework/framework/Constraints/Comparers/StringsComparer.cs

```diff
@@ -17,12 +18,12 @@ public static EqualMethodResult Equal(object x, object y, ref Tolerance toleranc
             if (tolerance.HasVariance)
                 return EqualMethodResult.ToleranceNotSupported;
 
-            return Equals(xString, yString, equalityComparer.IgnoreCase, equalityComparer.IgnoreWhiteSpace) ?
+            return Equals(xString, yString, equalityComparer.IgnoreCase, equalityComparer.IgnoreWhiteSpace, equalityComparer.NormalizeLineEndings) ?
                 EqualMethodResult.ComparedEqual :
                 EqualMethodResult.ComparedNotEqual;
         }
 
-        public static bool Equals(string x, string y, bool ignoreCase, bool ignoreWhiteSpace)
+        public static bool Equals(string x, string y, bool ignoreCase, bool ignoreWhiteSpace, bool NormalizeLineEndings)
```

Comment:
Use camelCase for parameter names.
Did this not raise IDE1006 for you?
![{93D7E055-CC6C-4BFC-A857-751090DD24EC}](https://github.com/user-attachments/assets/4d26cdb1-da13-42b9-8251-cc6702940761)